code/src/components/Details.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import { useParams, Link } from 'react-router-dom';
+import { NotFound } from './NotFound';
+
+export const Details = () => {
+  const [movieDetails, setMovieDetails] = useState([])
+
+  const { id } = useParams()
+
+  /* To round rating number to one decimal */
+  const roundedNumber = Math.round(movieDetails.vote_average * 10) / 10
```

Comment:
Nice that you got the round up to work! Like we mentioned on Monday, we tried adding the toFixed but that just made the site crash 😮